### ISSUE TITLE
gradle clean remove files checked into the repository

### ISSUE THREAD

**Issue by mar9000** [JetBrains/MPS-extensions#156]
I suppose that `gradlew clean` would not alter the repository status, instead:

```bash
marco@lizard /opt/working-dir/MPS-extensions $ ./gradlew clean 
[snip]
marco@lizard /opt/working-dir/MPS-extensions $ git status
On branch master
Your branch is up-to-date with 'origin/master'.
[snip]
	deleted:    build/allScripts/build.xml
	deleted:    build/generated/allScripts/build.xml
```

After this gradle build does not work anymore.

**Comment by coolya** [comment]
I can't reproduce it on current master. I suspect it was fixed as part of #153 

Can you verify that on your side?

**Comment by mar9000** [comment]
Yes it seems closed:
```bash
MPS-extensions $ git status
On branch master
[snip]
MPS-extensions $ git pull
Already up-to-date.
MPS-extensions $ ./gradlew clean
[snip]
BUILD SUCCESSFUL in 0s
2 actionable tasks: 2 up-to-date
MPS-extensions $ git status
On branch master
Your branch is up-to-date with 'origin/master'.

nothing to commit, working directory clean
```
